### PR TITLE
Sfl 79

### PR DESCRIPTION
**Pull request by aniket-Webonise**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -153,6 +154,10 @@ dependencies {
     kapt "com.google.dagger:dagger-android-processor:$rootProject.dagger2Version"
     kapt "com.google.android:android:$rootProject.googleAndroid"
 
+    implementation 'com.github.bumptech.glide:glide:4.4.0'
```
Comment (tejasAlsi-Webonise) [inline app/build.gradle:157]: take versions to build.gradle(project)

File: app/src/main/java/com/sjana/fitness/view/feature/home/move/MoveAdapter.kt [hunk 2]
```diff
@@ -1,4 +1,69 @@
...
+
+    fun addMoveList(moveList: List<ContentModel>) {
+        mMoveList = moveList
+    }
+
+    //the class is hodling the list view
+    class ViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+
+        fun bindItems(contentList: ContentModel, context: Context) {
+            val textViewFocusArea = itemView.findViewById(R.id.focusAreaDesc) as TextView
```
Comment (tejasAlsi-Webonise) [inline app/src/main/java/com/sjana/fitness/view/feature/home/move/MoveAdapter.kt:32]: all this is not required refer planner adapter

File: app/src/main/java/com/sjana/fitness/view/feature/home/move/MoveFragment.kt [hunk 3]
```diff
@@ -1,20 +1,36 @@
...
+
 import javax.inject.Inject
 
-class MoveFragment :  BaseFragment<MoveViewModel>(), MoveNavigator,
+class MoveFragment : BaseFragment<MoveViewModel>(), MoveNavigator,
     View.OnClickListener, View.OnFocusChangeListener {
 
+    private lateinit var recyclerView: RecyclerView
+    private lateinit var linearLayoutManager: LinearLayoutManager
+    private lateinit var jsonParser: JsonResponseParser
```
Comment (tejasAlsi-Webonise) [inline app/src/main/java/com/sjana/fitness/view/feature/home/move/MoveFragment.kt:30]: remove this

File: app/src/main/res/drawable/ic_oval.xml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (tejasAlsi-Webonise) [inline app/src/main/res/drawable/ic_oval.xml:1]: remove this file its already there

File: app/src/main/res/drawable/ic_oval__notselected.xml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (tejasAlsi-Webonise) [inline app/src/main/res/drawable/ic_oval__notselected.xml:1]: remove this file its already there

File: app/src/main/res/layout/move_list_item.xml [hunk 6]
```diff
@@ -331,4 +332,4 @@
         </androidx.constraintlayout.widget.ConstraintLayout>
 
     </androidx.constraintlayout.widget.ConstraintLayout>
-</com.google.android.material.card.MaterialCardView>
+</com.google.android.material.card.MaterialCardView>
```
Comment (tejasAlsi-Webonise) [inline app/src/main/res/layout/move_list_item.xml:335]: EOL error

File: app/src/main/res/values/styles.xml [hunk 7]
```diff
@@ -49,4 +49,10 @@
         <item name="android:fontFamily">@font/verdana_bold</item>
         <item name="android:layout_marginTop">21dp</item>
     </style>
+
```
Comment (tejasAlsi-Webonise) [inline app/src/main/res/values/styles.xml:52]: remove this

Comment (aniket-Webonise) [reply app/src/main/res/values/styles.xml:52]: done

**Pull request merged**